catalog/video/catalog-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -331,13 +345,21 @@ public void updateCallback(Map<String, Object> properties) {
                 return;
             }
 
+            if (properties.containsKey(METATYPE_DISTANCE_TOLERANCE) && properties.get(
```

Comment:
Normally, yes. But in this case the value can be null, and checkMetaTypeClass indicates an error on null values.